exercises/concept/squeaky-clean/.docs/introduction.md

```diff
@@ -0,0 +1,17 @@
+# Introduction
```

Comment:
This file must be an exact copy of the concept introduction file (which you can modify in this PR if you want, of course)